Ansible-security.md

```diff
@@ -0,0 +1,10 @@
+
+1. Ansible Server:
+    - Where is your ansible server located? 
+    - Who has access to it? 
+    - How do you minimize the access to the ansible server to a limited number of users.
+2. Ansible Vault:
+    - Do you use ansible-vault to encrypt the ansible playbook? 
+    - Where is the ansible vault password stored?
+3. Is SSH allowed only from the specific source address to the ansible server?
+4. Is a HashiCorp’s Vault used for storing the secrets? How are secrets rotated?
```

Comment:
I think this should be more technology neutral, and not call out to Hashicorp Vault specifically. What we really want to know is (a) how secrets are stored and (b) how often they are rotated. So replace with those two questions instead. :)

Reply:
As suggested , edited the question.